lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -293,14 +293,10 @@ def missing_framework_version(manifest)
           missing_keys_validation_error([RUBY_TO_JSON[:framework_version]]) if manifest.framework_version.nil?
         end
 
-        def invalid_version_error(manifest, package)
+        def invalid_version_error(manifest)
           valid_to_serve = AppVersion::TO_BE_SERVED
           target_version = manifest.framework_version
 
-          if target_version == AppVersion::DEPRECATED
```

Comment:
we don't need this error anymore?

Reply:
this doesn't make sense anymore. see screenshot on PR description. this warning is the yellow line on the picture above.